extractor.go

```diff
@@ -17,6 +17,9 @@ const bearer = "bearer"
 // UserKey ...
 const UserKey = "user"
 
+// Algorithm used for signing
+var Alg = jwt.SigningMethodHS256.Alg()
+
 // TokenExtractor Attempts to extract the token on every request
 func TokenExtractor(logger log.Logger, secret interface{}) kithttp.RequestFunc {
```

Comment:
Sure, but is it worth it?  I would think that entails:
 - each service needs to pass that parameter
 - so each service needs to have an additional configuration parameter
 - and it should probably be added to consul
 - but it also needs to be added to each .env.example file for local testing